src/components/PageHeader/PageHeader.test.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { TextField } from '@material-ui/core';
+
+import PageHeader from './PageHeader';
+
+describe('PageHeader', () => {
+  let component;
+  beforeEach(() => {
+    component = shallow(<PageHeader />);
+  });
+
+  it('Does header have an image', () => {
```

Comment:
Syntax note - it's generally considered best practice to phrase "it" blocks with a phrase starting with "should", i.e. "should have a header image" and "should contain a search bar" rather than question-style syntax.